homeassistant/components/point/strings.json

```diff
@@ -1,29 +1,18 @@
 {
```

Comment:
With oauth come a lot of extra strings, check `weheat` for an example of those new ones

Reply:
I can't find any `weheat` integration.

Reply:
Do you mean that I should copy something like this from withings?

```json
  "config": {
    "step": {
      "pick_implementation": {
        "title": "[%key:common::config_flow::title::oauth2_pick_implementation%]"
      },
      "reauth_confirm": {
        "title": "[%key:common::config_flow::title::reauth%]",
        "description": "The Withings integration needs to re-authenticate your account"
      }
    },
    "error": {
      "already_configured": "[%key:common::config_flow::abort::already_configured_account%]"
    },
    "abort": {
      "authorize_url_timeout": "[%key:common::config_flow::abort::oauth2_authorize_url_timeout%]",
      "missing_configuration": "[%key:common::config_flow::abort::oauth2_missing_configuration%]",
      "already_configured": "Configuration updated for profile.",
      "no_url_available": "[%key:common::config_flow::abort::oauth2_no_url_available%]",
      "oauth_error": "[%key:common::config_flow::abort::oauth2_error%]",
      "oauth_timeout": "[%key:common::config_flow::abort::oauth2_timeout%]",
      "oauth_unauthorized": "[%key:common::config_flow::abort::oauth2_unauthorized%]",
      "oauth_failed": "[%key:common::config_flow::abort::oauth2_failed%]"
    },
    "create_entry": {
      "default": "Successfully authenticated with Withings."
    }
  },
  
```

Reply:
Oh, maybe your branch is too much out of date for weheat :P

Reply:
I've updated strings.json now.